runtime/src/main/java/io/quarkiverse/githubaction/runtime/GitHubEvent.java

```diff
@@ -159,34 +159,27 @@ public DynamicGraphQLClient getDynamicGraphQLClient() {
     }
 
     public GHRepository getGHRepository() {
-        var repo = gitHubRepository;
-        if (repo == null) {
+        GHRepository localGitHubRepository = gitHubRepository;
+        if (localGitHubRepository == null) {
```

Comment:
I followed the same pattern as for the other methods for the local variable name.